package.json

```diff
@@ -41,7 +41,7 @@
     "protobufjs": "^5.0.0",
     "ts-node": "^1.7.0",
     "tslint": "^4.0.0",
-    "typescript": "^2.1.0"
+    "typescript": "2.4.2"
```

Comment:
Again, I believe there was a breaking change specifically between 2.4.0 and 2.4.1, so I don't think `^2.4.0` is a good idea, but `~2.4.2` should work. That being said, the fact that they introduced a breaking change in a patch release seems like perhaps a good reason to not trust that future patch releases will be problem-free, and given this is the compiler, I'd be a bit more inclined to leave this pegged exactly than with RXJS. I don't have super-strong feelings either way, though. What do you prefer?